### ISSUE TITLE
[MDCSnackbar] Multiple Snackbars bug: displayed at the same time

### ISSUE THREAD

**Issue by damian-mariachi-team** [material-components/material-components-web#5310]
## Bug report
Several snackbars should be displayed one after the other.
At the moment they were displayed at the same time

### Actual behavior
For Example: 2 Snackbars were displayed at the same time.

### Expected behavior
For Example: 2 Snackbars should be displayed one after the other. 

### Your Environment:

| Software         | Version(s) |
| ---------------- | ---------- |
| MDC Web          | 4.0.0
| Browser          | Chrome Version 78.0.3904.108
| Operating System | Windows 10

**Comment by CyborgSemon** [comment]
Could you provide a [CodePen](https://codepen.io/abhiomkar/pen/gQWarJ) showing off this bug?

**Comment by damian-mariachi-team** [comment]
Hi Simon!

Hier one example:
https://codepen.io/damian-mariachi-team/pen/OJPXbVq

Unfortunately, CodePen does not work with external libraries.
Must be tested locally.

In this example, both sanckbars are displayed in parallel when you quickly click on "Snackbar 1" and "Snackbar 2" buttons.

**Comment by CyborgSemon** [comment]
Hey @damian-mariachi-team!
CodePen does support external libraries. If you fork the CodePen from the link I provided above you can use MDC inside CodePen (Or link the Javascript and CSS CDNs in the CodePen settings).

With your provided example, I cant seem to trigger this error. Try using the CodePen link I provided to make a MDC Snackbar example :)
Thanks!

**Comment by damian-mariachi-team** [comment]
Hi Simon!

I've updated my example:
https://codepen.io/damian-mariachi-team/pen/wvBWJPE

**Comment by allan-chen** [comment]
Hi Simon,

Thanks for raising the issue! Orchestration between different snackbars is currently not supported in MDC, as each MDCSnackbar only knows about itself. We suggest writing your own orchestration logic in the meantime while this feature is pending.

**Comment by damian-mariachi-team** [comment]
Hi Allan!

But in demo with react it works ... 
https://material-components.github.io/material-components-web-catalog/#/component/snackbar

How it is possible?

**Comment by allan-chen** [comment]
The demo does some custom [coordination](https://github.com/material-components/material-components-web-catalog/blob/master/src/SnackbarCatalog.js#L46) that ensures that at most one snackbar is showing at a time.

**Comment by damian-mariachi-team** [comment]
Hi Allan!

I have a question:
What happens to this ticket with the status "closed"?
Is that defined as a feature for later? Do I have to open the new ticket for this?

Thanks in advance

**Comment by allan-chen** [comment]
Hi Damian,

We can certainly turn this into a feature request! In the meantime, please feel free to use custom coordination logic to manage multiple snackbars. Thanks again for raising this issue!

**Comment by damian-mariachi-team** [comment]
Hi Allan!

Thanks a lot.
